ui/src/layouts/Header/MobileMenu.tsx

```diff
@@ -1,117 +1,141 @@
-import React, { useState } from 'react'
```

Comment:
We are going to have to do something about your linter, this entire file shouldn't have changed like this and it makes it hard to see what actually did change